tests/src/FunctionalJavascript/EmbedDialogTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_oembed\FunctionalJavascript;
+
+use Drupal\Core\Url;
+
+/**
+ * Tests the embed dialog.
+ */
+class EmbedDialogTest extends EmbedTestBase {
```

Comment:
Looks like this test still covers only the Media entity type. We have to cover the possibility to use other entity types.

Reply:
We don't need coverage of all the things with all entity types. This test in fact tests the embeddable view mode configuration stuff.